examples/tutorials/curation/plot_3_upload_a_model.py

```diff
@@ -0,0 +1,125 @@
+"""
+Upload a pipeline to Hugging Face Hub
+=====================================
+"""
+##############################################################################
+# In this tutorial we will upload a pipeline, trained in SpikeInterface, to the
+# `Hugging Face Hub <https://huggingface.co/>`_ (HFH). To do this, you need a HFH account.
+# If you do not want to make an account, you can simply share the model folder with colleagues.
+# There are also several ways to interaction with HFH: the way we propose here doesn't use
+# many of the tools ``skops`` and hugging face have developed such as the ``Card`` and
+# ``hub_utils``. Feel free to check those out `here <https://skops.readthedocs.io/en/stable/examples.html>`_.
+#
+# The plan is to make a folder with the following file structure
+#
+# .. code-block::
+#
+#     my_model_folder/
+#         my_model_name.skops
+#         model_info.json
+#         training_data.csv
+#         labels.csv
+#         metadata.json
+#
+# SpikeInterface doesn't require you to keep this folder structure, we just advise it as
+# best practice.
+#
+# If you've used SpikeInterface to train your model, you have already created such a folder,
+# containing everything except the ``metadata.json`` file. In this file, we suggest saving
+# any information which shows when a model is applicable (and when it is *not*). Taking
+# a model trained on mouse data and applying it to a primate is likely a bad idea (or a
+# great research paper!). And a model trained in tetrode data will have limited application
+# on a silconehigh density probe. Hence we suggest the following dictionary as a minimal
+# amount of information needed. Note that we format the metadata so that the information
+# in common with the NWB data format is consistent with it,
+#
+# .. code-block::
+#
+#     model_metadata = {
+#         "subject_species": "Mus musculus",
+#         "brain_area": "CA1",
+#         "probe":
```

Comment:
Maybe this should be `probes` with a list? I imagine one could build a model using data from multiple probes, e.g. NP1/NP2